### PR TITLE
[rc] Merge master; remove deprecated `allowFullscreen` from popovers

### PR DESCRIPTION
**Pull request by Blackbaud-SteveBrush**

### CODE REVIEW

**Comment by blackbaud-ado** [comment]
Preview build at [https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-rc-remove-deprecated-features](https://developer.blackbaud.com/skyux-popovers?_sp.spa=pr-rc-remove-deprecated-features)

File: src/app/public/modules/popover/popover.directive.ts [hunk 1]
```diff
@@ -73,7 +73,6 @@ export class SkyPopoverDirective implements OnInit, OnDestroy {
 
   /**
    * Specifies the user action that displays the popover.
-   * @deprecated The trigger type `mouseenter` will be removed in the next major version.
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/popover/popover.directive.ts:76]: Why did we just remove the `deprecated` annotation here but leave the property?

File: src/app/public/modules/popover/types/popover-trigger.ts [hunk 2]
```diff
@@ -1,5 +1,4 @@
 /**
  * Specifies the user action that displays the popover.
- * @deprecated Specifying a trigger type other than 'click' will not be supported in the next major version.
```
Comment (Blackbaud-AlexKingman) [inline src/app/public/modules/popover/types/popover-trigger.ts:3]: Why did we just remove the `deprecated` annotation here but leave the property?

File: src/app/public/modules/popover/popover.directive.ts (hunk 1 above)
Comment (Blackbaud-AlexKingman) [reply src/app/public/modules/popover/popover.directive.ts:76]: Disregard. I see this was a merge from master.

Comment (Blackbaud-SteveBrush) [reply src/app/public/modules/popover/popover.directive.ts:76]: Todd wanted us to revert the deprecation for hover-activated popovers

File: src/app/public/modules/popover/types/popover-trigger.ts (hunk 2 above)
Comment (Blackbaud-SteveBrush) [reply src/app/public/modules/popover/types/popover-trigger.ts:3]: Same as above.

**Pull request merged**